### PR TITLE
Added topic_id arg to find_groups()

### PR DESCRIPTION
**Pull request by ledell**
Add `topic_id` argument to `find_groups()` function: https://github.com/rladies/meetupr/issues/13.  Currently `topic_id` expects a single integer. (note: "R-Ladies" topic id is 1513883)

### CODE REVIEW

**Comment by ledell** [comment]
"R-Ladies" topic id is 1513883.  I still need to add a [function](https://github.com/rladies/meetupr/issues/44) that will make it easy to look up topic ids from a search term.

This is how it works:
```
meetups <- find_groups(topic_id = 1513883, api_key = api_key)
```

File: R/find_groups.R [hunk 1]
```diff
@@ -1,6 +1,7 @@
-#' Find meetup groups using text-based search
+#' Find meetup groups matching a search query
 #'
-#' @param text string used to filter groups
+#' @param text Character. Raw full text search query.
+#' @param topic_id  Integer. Topic ID.
```
Comment (ledell) [inline R/find_groups.R]:
```suggestion
#' @param topic_id  Integer. Meetup.com topic ID.
```

File: man/find_groups.Rd [hunk 2]
```diff
@@ -2,12 +2,15 @@
...
 \usage{
-find_groups(text = NULL, radius = "global", api_key = NULL)
+find_groups(text = NULL, topic_id = NULL, radius = "global",
+  api_key = NULL)
 }
 \arguments{
-\item{text}{string used to filter groups}
+\item{text}{Character. Raw full text search query.}
+
+\item{topic_id}{Integer. Topic ID.}
```
Comment (ledell) [inline man/find_groups.Rd]:
```suggestion
\item{topic_id}{Integer. Meetup.com topic ID.}
```

**Comment by ledell** [comment]
@maelle @hfrick I moved some to-do items to a new [issue](https://github.com/rladies/meetupr/issues/44) so I can just be done with this PR and merge this.

**Comment by maelle** [comment]
By the way where does one find topic IDs? Sorry I hadn't looked at this for a long time.

**Comment by ledell** [comment]
@maelle There is not an easy way.  The way I looked up the R-Ladies topic id was to type "R-Ladies" into this query and look at the returned topic id: https://secure.meetup.com/meetup_api/console/?path=/find/topics  
This is what it returns:
```
HTTP/1.1 200 success
{

    "0": {
        "id": 1513883,
        "name": "R-Ladies",
        "urlkey": "r-ladies",
        "group_count": 139,
        "member_count": 39741,
        "description": "Find out what's happening in R-Ladies Meetup groups around the world and start meeting up with the ones near you.",
        "lang": "en_US"
    },
    "1": {
        "id": 1387242,
        "name": "WHERE R U HEADED LADY CAN I COME TOO?",
        "urlkey": "where-r-u-headed-lady-can-i-come-too",
        "group_count": 0,
        "member_count": 0,
        "description": "Find out what's happening in WHERE R U HEADED LADY CAN I COME TOO? Meetup groups around the world and start meeting up with the ones near you.",
        "lang": "en_US"
    }

}
```


Hopefully we can make this easier by creating a new `find_topics()` function: https://github.com/rladies/meetupr/issues/44  At the very least, we can parse this info into a tibble and the user can decide which topic id is appropriate by viewing the results (in the case where it's not just one result, like here).

**Pull request merged**